dds/DCPS/InfoRepoDiscovery/DataReaderRemoteImpl.cpp

```diff
@@ -38,12 +38,12 @@ DataReaderRemoteImpl::add_association(const RepoId& yourId,
                                       bool active)
 {
   if (DCPS_debug_level) {
-    GuidConverter writer_converter(yourId);
-    GuidConverter reader_converter(writer.writerId);
+    LogGuid writer_converter(yourId);
+    LogGuid reader_converter(writer.writerId);
```

Comment:
It might be appropriate to change the names of the variables since these are no longer GuiCoverter instances but LogGuid, some thing like `writer_log` might be a better name.

Reply:
It has been updated with better variable names.